core/src/main/kotlin/io/customer/sdk/core/di/AndroidSDKComponent.kt

```diff
@@ -21,19 +21,19 @@ class AndroidSDKComponent(
     val client: Client
 ) : DiGraph() {
     val applicationContext: Context
-        get() = newInstance { context.applicationContext }
+        get() = newInstance<Context> { context.applicationContext }
```

Comment:
is it needed as both (context and app context) are same type?

Reply:
Yes we can, updated